misc/scripts/build.sh

```diff
@@ -140,6 +139,9 @@ else
     -e "ESX=$2" \
     -e "VM1=$3" \
     -e "VM2=$4" \
+    -e "DS1=$DS1" \
+    -e "VM1NAME=$VM1NAME" \
```

Comment:
We have already introduced govc util to retrieve VM name from IP. So these variables are not necessary.